File: steps/2/test/java/eightyDays/java8/IdentificationTest.java
package eightyDays.java8;

import org.junit.Test;

import static org.junit.Assert.*;

public class IdentificationTest {
    @Test
    public void testGetNumber() throws Exception {
        Identification out = new Identification();
        assertTrue(out.getNumber().toString().length() > 0);
        assertTrue(out.getNumber() != new Identification().getNumber());
    }
}

File: steps/7/test/scala/eightyDays/scala/IdentificationTest.scala
package eightyDays.scala

import org.scalatest.WordSpec

class IdentificationTest extends WordSpec {
  val have = afterWord("have")
  "A new identification" must have {
    "a unique number" in {
      val out = Identification()
      assert(out.number.toString.length > 0)
      assert(out.number !== Identification().number)
    }
  }
}